tests/integration-tests/tests/gb200/test_gb200.py

```diff
@@ -0,0 +1,247 @@
+         "status": "CONNECTED",
+         "changed": true
+        }
+       },
+       "changed": true,
+       "version": "570.172.08"
+      }
+     },
+     "timestamp": "8/8/2025 17:38:02.641",
+     "status": "UP"
+    }
+    """
+    slurm = SlurmCommands(rce)
+
+    imex_statuses = []
+    for reporting_node_ip in ips:
+        reporting_node_name = slurm.get_nodename_from_ip(reporting_node_ip)
+        logging.info(f"Retrieving IMEX status reported by node {reporting_node_ip} with hostname {reporting_node_name}")
+        result_stdout = rce.run_remote_command(f"cat result_{job_id}_{reporting_node_name}.out").stdout
+        result_stderr = rce.run_remote_command(f"cat result_{job_id}_{reporting_node_name}.err").stdout
+        assert_that(result_stderr).is_empty()
+        logging.info(
+            f"IMEX status reported by node {reporting_node_ip} with hostname {reporting_node_name}: {result_stdout}"
+        )
+        imex_statuses.append(json.loads(result_stdout))
+    latest_imex_status = max(imex_statuses, key=lambda i: datetime.strptime(i["timestamp"], "%m/%d/%Y %H:%M:%S.%f"))
+    logging.info(f"Latest IMEX status: {latest_imex_status}")
+    logging.info(f"Checking that IMEX is able to interconnect the expected nodes according to the latest status")
+    assert_that(latest_imex_status["status"]).is_equal_to("UP")
+    for ip_source in ips:
+        node_item = next(filter(lambda i: i["host"] == ip_source, latest_imex_status["nodes"].values()), None)
+        assert_that(node_item).is_not_none()
+        assert_that(node_item["status"]).is_equal_to("READY")
+        for ip_destination in ips:
+            connection_item = next(
+                filter(lambda i: i["host"] == ip_destination, node_item["connections"].values()), None
+            )
+            assert_that(node_item).is_not_none()
+            assert_that(connection_item["status"]).is_equal_to("CONNECTED")
+
+
+def assert_imex_healthy(cluster: Cluster, queue_name: str, compute_resource_name: str, max_nodes: int = 1):
+    rce = RemoteCommandExecutor(cluster)
+
+    launch_template_id = cluster.get_compute_nodes_launch_template_logical_id(queue_name, compute_resource_name)
+    logging.info(
+        f"Launch template for nodes in queue {queue_name} and compute resource {compute_resource_name}: "
+        f"{launch_template_id}"
+    )
+
+    ips = cluster.get_compute_nodes_private_ip(queue_name, compute_resource_name)
+    logging.info(
+        f"Private IP addresses for nodes in queue {queue_name} and compute resource {compute_resource_name}: " f"{ips}"
+    )
+
+    job_id = submit_job_imex_status(rce, launch_template_id, queue_name, max_nodes)
+
+    assert_imex_nodes_config_is_correct(rce, launch_template_id, ips)
+    assert_imex_status_ok(rce, job_id, ips)
+    assert_no_errors_in_logs(cluster, queue_name, compute_resource_name)
+
+
+def assert_imex_not_configured(cluster: Cluster, queue_name: str, compute_resource_name: str, max_nodes: int = 1):
```

Comment:
If we use the ExtraChefAttribute as per https://github.com/aws/aws-parallelcluster-cookbook/pull/3001 it would apply to these nodes too. I think we should add a check in terms of `assert_imex_status_is_not_ok` w.r.t at least that the status if NOT UP or is DOWN. 

If the prolog was extended to run on these node then the status would be DOWN but the `nvidia-imex-ctl` would be running.

Reply:
Added